testing ubuntu prb again

shaneknapp opened the pull request:
## What changes were proposed in this pull request?

(Please fill in changes proposed in this fix)

## How was this patch tested?

(Please explain how this patch was tested. E.g. unit tests, integration tests, manual tests)
(If this patch involves UI changes, please attach a screenshot; otherwise, remove this)

Please review http://spark.apache.org/contributing.html before opening a pull request.

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

shaneknapp committed:
testing ubuntu prb again

AmplabJenkins commented:
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
https://amplab.cs.berkeley.edu/jenkins//job/ubuntuSparkPRB/62/
Test FAILed.

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp committed:
pystyle

shaneknapp commented:
test this please

AmplabJenkins commented:
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
https://amplab.cs.berkeley.edu/jenkins//job/ubuntuSparkPRB/63/
Test FAILed.

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp committed:
reading error messages is hard

shaneknapp commented:
test this please

AmplabJenkins commented:
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
https://amplab.cs.berkeley.edu/jenkins//job/ubuntuSparkPRB/64/
Test PASSed.

shaneknapp commented:
test this please

AmplabJenkins commented:
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
https://amplab.cs.berkeley.edu/jenkins//job/ubuntuSparkPRB/65/
Test FAILed.

shaneknapp commented:
test this please

shaneknapp committed:
bumping python to 3.5

shaneknapp committed:
bumping python to 3.5

shaneknapp committed:
bumping python to 3.5

shaneknapp committed:
bumping python to 3.5

shaneknapp committed:
bumping python to 3.5

shaneknapp committed:
bumping python to 3.5

shaneknapp committed:
bumping python to 3.5

shaneknapp committed:
bumping python to 3.5

shaneknapp committed:
bumping python to 3.5

shaneknapp committed:
bumping python to 3.5

shaneknapp committed:
bumping python to 3.5

shaneknapp committed:
bumping python to 3.5

shaneknapp committed:
bumping python to 3.5

shaneknapp committed:
bumping python to 3.5

shaneknapp committed:
bumping python to 3.5

shaneknapp committed:
bumping python to 3.5

shaneknapp committed:
bumping python to 3.5

shaneknapp committed:
bumping python to 3.5

shaneknapp committed:
bumping python to 3.5

shaneknapp committed:
bumping python to 3.5

shaneknapp committed:
bumping python to 3.5

shaneknapp committed:
bumping python to 3.5

shaneknapp committed:
bumping python to 3.5

shaneknapp committed:
bumping python to 3.5

shaneknapp committed:
bumping python to 3.5

shaneknapp committed:
bumping python to 3.5

AmplabJenkins commented:
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
https://amplab.cs.berkeley.edu/jenkins//job/ubuntuSparkPRB/66/
Test FAILed.

AmplabJenkins commented:
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
https://amplab.cs.berkeley.edu/jenkins//job/ubuntuSparkPRB/67/
Test FAILed.

shaneknapp commented:
test this please

AmplabJenkins commented:
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
https://amplab.cs.berkeley.edu/jenkins//job/ubuntuSparkPRB/68/
Test PASSed.

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

shaneknapp committed:
testing symlink of python3.4 -> python3.5

AmplabJenkins commented:
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
https://amplab.cs.berkeley.edu/jenkins//job/ubuntuSparkPRB/69/
Test FAILed.

shaneknapp commented:
test this please

AmplabJenkins commented:
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
https://amplab.cs.berkeley.edu/jenkins//job/ubuntuSparkPRB/70/
Test PASSed.

shaneknapp committed:
testing new python 3.5 env

shaneknapp committed:
testing new python 3.5 env

AmplabJenkins commented:
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
https://amplab.cs.berkeley.edu/jenkins//job/ubuntuSparkPRB/71/
Test PASSed.

shaneknapp committed:
test the proper python version

shaneknapp committed:
test the proper python version

shaneknapp committed:
test the proper python version

shaneknapp committed:
test the proper python version

shaneknapp committed:
test the proper python version

shaneknapp committed:
test the proper python version

shaneknapp committed:
test the proper python version

shaneknapp committed:
test the proper python version

shaneknapp committed:
test the proper python version

shaneknapp committed:
test the proper python version

shaneknapp committed:
test the proper python version

shaneknapp committed:
test the proper python version

shaneknapp committed:
test the proper python version

shaneknapp committed:
test the proper python version

shaneknapp committed:
test the proper python version

shaneknapp committed:
test the proper python version

shaneknapp committed:
test the proper python version

shaneknapp committed:
test the proper python version

shaneknapp committed:
test the proper python version

shaneknapp committed:
test the proper python version

shaneknapp committed:
test the proper python version

shaneknapp committed:
test the proper python version

shaneknapp committed:
test the proper python version

shaneknapp committed:
test the proper python version

AmplabJenkins commented:
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
https://amplab.cs.berkeley.edu/jenkins//job/ubuntuSparkPRB/72/
Test FAILed.

shaneknapp commented:
test this please

AmplabJenkins commented:
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
https://amplab.cs.berkeley.edu/jenkins//job/ubuntuSparkPRB/73/
Test PASSed.

shaneknapp commented:
test this please

AmplabJenkins commented:
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
https://amplab.cs.berkeley.edu/jenkins//job/ubuntuSparkPRB/74/
Test FAILed.

shaneknapp commented:
test this please

shaneknapp committed:
Merge branch 'ubuntu-testing' of github.com:shaneknapp/spark into ubuntu-testing

shaneknapp commented:
test this please

ifilonenko committed:
[SPARK-23984][K8S] Changed Python Version config to be camelCase

## What changes were proposed in this pull request?

Small formatting change to have Python Version be camelCase as per request during PR review.

## How was this patch tested?

Tested with unit and integration tests

Author: Ilan Filonenko <if56@cornell.edu>

Closes #22095 from ifilonenko/spark-py-edits.

AmplabJenkins commented:
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
https://amplab.cs.berkeley.edu/jenkins//job/ubuntuSparkPRB/75/
Test FAILed.

AmplabJenkins commented:
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
https://amplab.cs.berkeley.edu/jenkins//job/ubuntuSparkPRB/76/
Test FAILed.

Commit:
[SPARK-25082][SQL] improve the javadoc for expm1()

## What changes were proposed in this pull request?
Correct the javadoc for expm1() function.

## How was this patch tested?
None. It is a minor issue.

Closes #22115 from bomeng/25082.

Authored-by: Bo Meng <bo.meng@jd.com>
Signed-off-by: hyukjinkwon <gurwls223@apache.org>

codeatri committed:
[SPARK-23939][SQL] Add transform_keys function

## What changes were proposed in this pull request?
This pr adds transform_keys function which applies the function to each entry of the map and transforms the keys.
```javascript
> SELECT transform_keys(map(array(1, 2, 3), array(1, 2, 3)), (k,v) -> k + 1);
       map(2->1, 3->2, 4->3)

> SELECT transform_keys(map(array(1, 2, 3), array(1, 2, 3)), (k,v) -> k + v);
       map(2->1, 4->2, 6->3)
```

## How was this patch tested?
Added tests.

Closes #22013 from codeatri/SPARK-23939.

Authored-by: codeatri <nehapatil6@gmail.com>
Signed-off-by: Takuya UESHIN <ueshin@databricks.com>

techaddict committed:
[SPARK-23932][SQL] Higher order function zip_with

## What changes were proposed in this pull request?
Merges the two given arrays, element-wise, into a single array using function. If one array is shorter, nulls are appended at the end to match the length of the longer array, before applying function:
```
    SELECT zip_with(ARRAY[1, 3, 5], ARRAY['a', 'b', 'c'], (x, y) -> (y, x)); -- [ROW('a', 1), ROW('b', 3), ROW('c', 5)]
    SELECT zip_with(ARRAY[1, 2], ARRAY[3, 4], (x, y) -> x + y); -- [4, 6]
    SELECT zip_with(ARRAY['a', 'b', 'c'], ARRAY['d', 'e', 'f'], (x, y) -> concat(x, y)); -- ['ad', 'be', 'cf']
    SELECT zip_with(ARRAY['a'], ARRAY['d', null, 'f'], (x, y) -> coalesce(x, y)); -- ['a', null, 'f']
```
## How was this patch tested?
Added tests

Closes #22031 from techaddict/SPARK-23932.

Authored-by: Sandeep Singh <sandeep@techaddict.me>
Signed-off-by: Takuya UESHIN <ueshin@databricks.com>

shaneknapp committed:
Merge branch 'ubuntu-testing' of github.com:shaneknapp/spark into ubuntu-testing

AmplabJenkins commented:
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
https://amplab.cs.berkeley.edu/jenkins//job/ubuntuSparkPRB/77/
Test FAILed.
